src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CreateCohortWithDraftApprenticeshipController.cs

```diff
@@ -89,6 +91,22 @@ public async Task<IActionResult> AddDraftApprenticeship(AddDraftApprenticeshipVi
             }
         }
 
+        [HttpGet]
+        [Route("add-select-employer")]
```

Comment:
I think this may have been added in confusion due to similar routes in the pre-existing pages. I don't think the `add-` one serves a purpose?